providers/implementations/digests/blake2_prov.c

```diff
@@ -95,8 +95,8 @@ static int blake2b512_get_params(OSSL_PARAM params[])
     return ossl_digest_default_get_params(params, BLAKE2B_BLOCKBYTES, 64, 0);
 }
 
-const OSSL_DISPATCH ossl_blake2b512_functions[] =
-    { {OSSL_FUNC_DIGEST_NEWCTX, (void (*)(void))blake2b512_newctx},
+const OSSL_DISPATCH ossl_blake2b512_functions[] = {
+    {OSSL_FUNC_DIGEST_NEWCTX, (void (*)(void))blake2b512_newctx},
```

Comment:
Hmm... you could also add a space after `{` here and on below lines.

Reply:
I could indeed, but `checkpatch.pl` doesn't report an issue, at least not yet, not in the classes of issues I have been addressing so far.

Running `checkpatch.pl` on the whole code base takes hours, so I've run it once in August. I'll re-run the script to catch new or unfixed issues once I am finished going through the current ones.